src/planwise/client/projects2/handlers.cljs

```diff
@@ -152,3 +152,27 @@
  (fn [db [_ projects-list]]
    (assoc db :list projects-list)))
 
+;;------------------------------------------------------------------------------
+;; Filtering by tags
+
+(rf/reg-event-fx
+ :projects2/save-tag
+ in-projects2
+ (fn [{:keys [db]} [_ tag]]
+   (let [path [:current-project :config :sites :tags]
+         n (count (get-in db path))]
+     {:db          (if (pos? n)
+                     (assoc-in db (into path [n]) tag)
+                     (assoc-in db path [tag]))
+      :dispatch [:projects2/persist-current-project]})))
+
+(rf/reg-event-fx
+ :projects2/delete-tag
+ in-projects2
+ (fn [{:keys [db]} [_ index]]
+   (let [path [:current-project :config :sites :tags]
+         tags* (vec (keep-indexed #(if (not= %1 index) %2) (get-in db path)))]
```

Comment:
This (remove vector element) looks useful enough to extract as a function into `planwise.client.utils`.